easybuild/easyconfigs/s/ScaLAPACK/ScaLAPACK-2.0.2-gompic-2016.08-OpenBLAS-0.2.15-LAPACK-3.6.0.eb

```diff
@@ -0,0 +1,25 @@
+name = 'ScaLAPACK'
+version = '2.0.2'
+
+homepage = 'http://www.netlib.org/scalapack/'
+description = """The ScaLAPACK (or Scalable LAPACK) library includes a subset of LAPACK routines
+ redesigned for distributed memory MIMD parallel computers."""
+
+toolchain = {'name': 'gompic', 'version': '2016.08'}
+toolchainopts = {'pic': True}
+
+source_urls = [homepage]
+sources = [SOURCELOWER_TGZ]
+
+blaslib = 'OpenBLAS'
+blasver = '0.2.15'
+blassuff = '-LAPACK-3.6.0'
+
+versionsuffix = "-%s-%s%s" % (blaslib, blasver, blassuff)
+
+dependencies = [(blaslib, blasver, blassuff, ('GCC', '4.9.3-2.25'))]
```

Comment:
@eshelman tests fail because of this hardcoded toolchain spec for OpenBLAS

First, it's wrong, since the easyconfig for OpenBLAS you provide uses `gompic`; also, there's no need to hardcode a toolchain spec at all in this case...

Any particular reason you added it?